tests/testthat/helper.R

```diff
@@ -40,29 +40,34 @@ expect_meta_match <- function(file, core = "occurrence.csv", ...) {
 
   # Parse reference meta.xml from inst/extdata/meta.xml
   xml_list <-
-    xml2::read_xml(system.file("extdata", "meta.xml", package = "movepub")) %>%
+    xml2::read_xml(system.file("extdata", "meta.xml", package = "movepub")) |>
     xml2::as_list()
   xml_file_location <-
-    purrr::chuck(xml_list, "archive", core_or_extension, "files", "location") %>%
+    purrr::chuck(xml_list, "archive", core_or_extension, "files", "location") |>
     unlist()
   xml_file_fields <-
-    xml_list %>%
-    purrr::chuck("archive", core_or_extension) %>%
+    xml_list |>
+    purrr::chuck("archive", core_or_extension) |>
     purrr::map_dfr(~ dplyr::tibble(
       index = as.numeric(attr(.x, which = "index")),
       term = attr(.x, which = "term")
-    )) %>%
-    dplyr::filter(!is.na(term)) %>%
+    )) |>
+    dplyr::filter(!is.na(term)) |>
     dplyr::mutate(field = basename(term), .keep = "unused")
 
   # Get fields from csv
   csv_file_cols <-
-    readr::read_csv(file, show_col_types = FALSE) %>%
-    colnames() %>%
+    readr::read_csv(file, show_col_types = FALSE) |>
+    colnames() |>
     purrr::map_chr(~ sub("^[A-Za-z]+:", "", .x)) # Remove namespace like "dcterms:"
   csv_file_fields <-
-    dplyr::tibble(field = csv_file_cols) %>%
-    dplyr::mutate(index = as.integer(rownames(.)) - 1, .before = field) # Add index
+    dplyr::tibble(field = csv_file_cols) |>
+    (\(df) {
+      dplyr::mutate(df,
+        index = as.integer(rownames(df)) - 1,
+        .before = field
```

Comment:
```suggestion
        .before = dplyr::all_of("field")
```

We might want to use tidyselect here, it'll avoid a lintr hit and it'll increase defensiveness. 

In this case it seems pretty unbelievable that the field column would not exist if it's only initiated a few lines above, but in general it's pretty solid advice.